qtdeclarative.pro

```diff
@@ -1,3 +1,10 @@
 CONFIG += tests_need_tools
 load(qt_parts)
 
+CONFIG(release, debug|release) {
```

Comment:
ok great, but what will provide the configuration for building release vs debug packages in the spec?